apps/sms/style/sms.css

```diff
@@ -829,6 +829,12 @@ Do not remove.
   background: #fff;
 }
 
+/* Adjust the name line-height to prevent bottom of chars been cut */
+.contact-list .suggestion > .name {
+  line-height: 2.4rem;
+  margin-top: -0.2rem;
```

Comment:
Since increasing the line-height will stretch the space a little bit, I have to set the negtive margin to make the name to keep in the original position, otherwise the space above the name span will be a little bit wider than original one.